src/OpenLoco/Vehicles/VehicleHead.cpp

```diff
@@ -25,6 +27,11 @@ using namespace OpenLoco::Literals;
 namespace OpenLoco::Vehicles
 {
     static loco_global<string_id, 0x009C68E6> gGameCommandErrorText;
+    static loco_global<uint8_t, 0x009C68EA> gGameCommandExpenditureType; // premultiplied by 4
+    static loco_global<int16_t, 0x009C68E0> gameCommandMapX;
+    static loco_global<int16_t, 0x009C68E2> gameCommandMapY;
+    static loco_global<int16_t, 0x009C68E4> gameCommandMapZ;
```

Comment:
Any chance we could access these with getter/setters functions instead? It will be harder to integrate these variables eventually when they are defined as `loco_globals` in several places.